cmd/renderer_test.go

```diff
@@ -128,6 +129,28 @@ func TestRendererTable_RenderBridgeList(t *testing.T) {
 	}
 }
 
+func TestRendererTable_Render_TxAttempts(t *testing.T) {
+	t.Parallel()
+
+	attempts := []models.TxAttempt{
+		models.TxAttempt{
+			Hash:      cltest.NewHash(),
+			TxID:      1,
+			GasPrice:  big.NewInt(1),
+			Confirmed: false,
+			Hex:       "0xdeadbeef",
+			SentAt:    1,
+		},
+	}
+
+	buffer := bytes.NewBufferString("")
+	r := cmd.RendererTable{Writer: buffer}
+
+	assert.NoError(t, r.Render(&attempts))
+	output := buffer.String()
+	assert.Contains(t, output, attempts[0].Hash.Hex())
```

Comment:
Sure, I'll add that. My personal style is to keep presentation assertions loose so that we can make changes to the table and implementation without always having to churn on the test. I figured the hex should always be there. But I'm not strongly attached to that.